src/main/scala/com/gu/liveappversions/S3Storage.scala

```diff
@@ -2,28 +2,35 @@ package com.gu.liveappversions
 
 import java.io.ByteArrayInputStream
 
+import com.amazonaws.AmazonServiceException
 import com.amazonaws.services.s3.AmazonS3ClientBuilder
 import com.amazonaws.services.s3.model.{ CannedAccessControlList, ObjectMetadata, PutObjectRequest, PutObjectResult }
 import com.gu.config.Aws
 import com.gu.config.Config.Env
-import com.gu.liveappversions.ios.BuildOutput
 import com.gu.liveappversions.ios.Lambda.logger
+import io.circe.Json
 import io.circe.syntax._
 
+import scala.io.Source
 import scala.util.{ Failure, Success, Try }
 
-object S3Uploader {
+object S3Storage {
 
   private val s3Client = AmazonS3ClientBuilder.standard()
     .withCredentials(Aws.credentials("developerPlayground"))
     .withRegion(Aws.euWest1.getName)
     .build()
 
-  def attemptUpload(buildOutput: BuildOutput, env: Env, bucketName: String): Try[PutObjectResult] = {
-
+  def fullKey(env: Env, partialKey: String): String = {
     val stagePrefix = if (env.stage == "PROD") "/" else s"/${env.stage}/"
-    val fileObjectKeyName = s"reserved-paths${stagePrefix}ios-live-app/recent-beta-releases.json"
```

Comment:
Like above, are these not currently read by iOS clients?

Reply:
Yep, but since it's iOS specific it's been [moved](https://github.com/guardian/live-app-versions/pull/24/files#diff-db7d0ed9f1bce06af9a47faa2b2b5823R28) out of this (shared) object and into the iOS lambda code.